notebooks/templates/Inspection_jobs_analysis_TEMPLATE.ipynb

```diff
@@ -7,7 +7,7 @@
    },
```

Comment:
<p><code>f"Interpolated IQR for job_duration for different inspection batches"</code> doesn't have to be an f-string</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thoth-station/notebooks/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>